### DISCUSSION TITLE
just ignore it #2

### DISCUSSION THREAD

**Discussion by ghost** [community/community#125181] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

{
  "liveServer.settings.donotShowInfoMsg": true,
  "explorer.confirmDelete": false,
  "editor.fontFamily": "JetBrains Mono",
  "editor.lineHeight": 1.5,
  "editor.fontLigatures": true,
  "workbench.editor.labelFormat": "short",
  "editor.semanticHighlighting.enabled": false,

  "[python]": {
    "editor.formatOnType": true,
    "editor.defaultFormatter": "ms-python.black-formatter"
  },
  "editor.fontSize": 14,
  "python.formatting.provider": "none",
  "workbench.iconTheme": "vscode-icons",
  "git.enableSmartCommit": true,
  "git.autofetch": true,
  "git.confirmSync": false,
  "liveServer.settings.donotVerifyTags": true,
  "workbench.startupEditor": "newUntitledFile",
  "keyboard.layout": "00000416",
  "editor.wordWrap": "on",
  "files.autoSave": "afterDelay",
  "terminal.integrated.defaultProfile.windows": "Git Bash",
  "[javascript]": {
    "editor.defaultFormatter": "vscode.typescript-language-features"
  },
  "[html]": {
    "editor.defaultFormatter": "vscode.html-language-features"
  },
  "[json]": {
    "editor.defaultFormatter": "vscode.json-language-features"
  },
  "[css]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
  "git.openRepositoryInParentFolders": "always",
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "database-client.capitalizeKeywordsWhenFormatting": true,
  "explorer.confirmDragAndDrop": false,
  "terminal.integrated.fontFamily": "monospace",
  "[typescriptreact]": {
    "editor.defaultFormatter": "vscode.typescript-language-features"
  },
  "editor.formatOnSave": true,
  "workbench.colorTheme": "Bearded Theme Arc Reversed"
}

**Comment by cixpy** [comment, marked as answer]
congrats
